_variables.tf

```diff
@@ -284,4 +284,10 @@ variable "publicly_accessible" {
   description = "(Optional) Bool to control if instance is publicly accessible"
   type        = bool
   default     = false
+}
+
+variable "license_model" {
+  description = "License model information for this DB instance (Optional, but required for some DB engines, i.e. Oracle SE1 and SQL Server)"
+  type        = string
+  default     = ""
```

Comment:
This default changes current default of **aws_db_instance**:

https://github.com/terraform-aws-modules/terraform-aws-rds/blob/master/modules/db_instance/variables.tf#L45

I believe the should match

Reply:
Changed default to `null`.